relnotes/closures.feature.md

```diff
@@ -0,0 +1,5 @@
+### Fixup of swarm internals to prevent closure allocation
```

Comment:
A feature release note in a patch release?

Reply:
Yes, we do it occasionally when fixing something implies adding new feature behind the scene. Though here maybe it is more appropriate to just mark it as `bug.md`

Reply:
Updated

Reply:
Mm, still looks like ` relnotes/closures.feature.md` to me.

Reply:
I don't see this as a feature, but as a bugfix for two reasons:

- it doesn't impose anything to the user API-wise
- it makes swarm to work as expected.

I don't think this actually needs any release note.